config/prometheus/monitor.yaml

```diff
@@ -1,20 +1,24 @@
 
-# Prometheus Monitor Service (Metrics)
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  labels:
-    control-plane: controller-manager
-  name: controller-manager-metrics-monitor
+  name: controller-manager
```

Comment:
this is a pretty generic name for the `system` namespace
we can getaway with `controller-manager` generic name ONLY if we are in a numaresources namespace

Reply:
But the `system` namespace will always be replaced with `numaresources`.
Do you wish to return to `controller-manager-metrics-monitor`?

Reply:
ok, if this is gonna be sitting in the `numaresources` namespace, it's good